README.md

```diff
@@ -157,6 +157,9 @@ You'll need to upload these to an s3 bucket, then you can use these files to
 [create a simulation application](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-application.html), 
 and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-job.html) in RoboMaker.
 
+## Generate Occupancy Map via map generation plugin
+Note: Procedural map generation is currently not supported in ROS2 (Dashing).
```

Comment:
I feel that we should say "... is only supported in ROS1 Kinetic and Melodic as we only tested in these two distribution.

Reply:
ros1 branch already states "ROS Kinetic / ROS Melodic - Other versions may work, however they have not been tested" in README requirements

Reply:
I see, thanks for the clarification

Reply:
also to clarify, we did test in ROS2 Dashing and plugin dependency is not supported => no map generation support in ROS2 Dashing